sql/files/defaultdata/rs/run

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+DEST="$1" ; shift
+MEMLIMIT="$1" ; shift
+
+# first you need to ./make_ch_root -i rustc
```

Comment:
I don't think we add this to the other scripts, so we should consider to not add it here.

Reply:
i saw it in the cpp and c scripts

Reply:
https://github.com/DOMjudge/domjudge/blob/main/sql/files/defaultdata/cpp/run

Reply:
Ok, 
```suggestion
# first you need to ./dj_make_chroot -i rustc
```